packages/split-view/README.md

```diff
@@ -0,0 +1,156 @@
+## Description
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/split-view?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/split-view)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/split-view?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/split-view)
+
+```
+yarn add @spectrum-web-components/split-view
+```
+
+Import the side effectful registration of `<sp-split-view>` via:
+
+```
+import '@spectrum-web-components/split-view/sp-split-view.js';
+```
+
+When looking to leverage the `SplitView` base class as a type and/or for extension purposes, do so via:
+
+```
+import { SplitView } from '@spectrum-web-components/split-view';
+```
+
+### Variants
+
+#### Horizontal
+
+```html
+<sp-split-view>
+    <div slot="primary">Left panel</div>
```

Comment:
Make sure we update this to the all default slot API.

Reply:
Done, removed it from README, stories and tests.